file_paths.md

```diff
@@ -275,6 +279,12 @@ create a `puppet` user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        bin *
+            pxp-agent.exe *
+        pxp-modules *
+            pxp-module-puppet *
```

Comment:
This pathing structure seems really redundant to me - `pxp-agent\pxp-modules\pxp-module-puppet`.  But perhaps that's by design?